### PR TITLE
sort: add --general-numeric-sort example

### PR DESCRIPTION
**Pull request by hntee**
- [ ] The page (if new), does not already exist in the repo.
- [x] The page is in the correct platform directory (`common/`, `linux/`, etc.)
- [x] The page has 8 or fewer examples.
- [x] The PR title conforms to the recommended [templates](/tldr-pages/tldr/blob/master/CONTRIBUTING.md#commit-message).
- [x] The page follows the [content guidelines](/tldr-pages/tldr/blob/master/CONTRIBUTING.md#guidelines).
- [x] The page description includes a link to documentation or a homepage (if applicable).

### CODE REVIEW

File: pages/common/sort.md [hunk 1]
```diff
@@ -29,3 +29,7 @@
 - Sort human-readable numbers (in this case the 5th field of `ls -lh`):
 
 `ls -lh | sort -h -k 5`
+
+- Sort numbers with exponents:
+
+`sort -g {{filename}}`
```
Comment (navarroaxel) [inline pages/common/sort.md:35]: we prefer the long argument:
```suggestion
`sort --general-numeric-sort {{path/to/file}}`
```
await confirmation before apply

**Review by navarroaxel — COMMENTED** [review]
Hey @hntee thanks for your contribution! I think this page is some outdated respect the style guide.

Could you add the link to the documentation in the page description?
```
> Sort lines of text files.
> More information: <https://www.gnu.org/software/coreutils/manual/html_node/sort-invocation.html>.
```

**Comment by hntee** [comment]
Hi @navarroaxel , thanks for your reply.
I have changed the parameter to the longer one and added the link.

Another question: since the above descriptions are all `{{filename}}`, is it better to modify all of these to `{{path/to/file}}`?

**Comment by navarroaxel** [comment]
> Hi @navarroaxel , thanks for your reply.
> I have changed the parameter to the longer one and added the link.

@hntee thanks!

> Another question: since the above descriptions are all `{{filename}}`, is it better to modify all of these to `{{path/to/file}}`?

I had that in mind, and also fix some arguments like `-r` to `--reverse` but I want the confirmation from @sbrl before apply those changes. Sometime there are exceptions for the rule.

**Review by sbrl — APPROVED** [review]
Thanks, @hntee!

Yeah since we're here, if you wouldn't mind adjusting all the other `{{filename}}` instances to `{{path/to/filename}}` that would be great!

Generally it's only pages like `diff` for example that are exceptions.

**Comment by hntee** [comment]
Hi @navarroaxel @sbrl , I have modified all the {{filename}} instances to {{path/to/filename}}

**Review by navarroaxel — APPROVED** [review]

**Comment by sbrl** [comment]
Thanks, @hntee :smiley_cat:

**Pull request merged**